circleguard/loadable.py

```diff
@@ -722,3 +722,8 @@ def __str__(self):
             return f"Loaded ReplayPath by {self.username} on {self.map_id} at {self.path}"
         else:
             return f"Unloaded ReplayPath at {self.path}"
+
+    def positionally_translate_replay(self):
+        noise = np.random.normal(0, 1, )
+        self.xy = self.xy + np.random.default_rng().multivariate_normal([10, 10], [[3,3],[3,3]],(len(self.xy)))
```

Comment:
What version of numpy are you running? This errors for me -

```python
Traceback (most recent call last):
  File "/Users/tybug/Desktop/coding/osu/circlecore/tests/test.py", line 72, in test_robustness_to_translation
    r = list(self.cg.steal_check(replays))
  File "/Users/tybug/Desktop/coding/osu/circlecore/circleguard/circleguard.py", line 151, in steal_check
    yield from self.run(loadables, Detect.STEAL, loadables2)
  File "/Users/tybug/Desktop/coding/osu/circlecore/circleguard/circleguard.py", line 107, in run
    comparer = Comparer(replays1, replays2=replays2)
  File "/Users/tybug/Desktop/coding/osu/circlecore/circleguard/comparer.py", line 50, in __init__
    replay.positionally_translate_replay()
  File "/Users/tybug/Desktop/coding/osu/circlecore/circleguard/loadable.py", line 728, in positionally_translate_replay
    self.xy = self.xy + np.random.default_rng().multivariate_normal([10, 10], [[3,3],[3,3]],(len(self.xy)))
AttributeError: module 'numpy.random' has no attribute 'default_rng'
```

I'm on `1.15.4`. I don't think we pin numpy to a minimum version in `setup.py`, but maybe we should.

Reply:
This is my pipfreeze:

```
circleparse==6.2.0
click==7.1.2
idna==2.9
numpy==1.18.3
ossapi==1.2.3
pytz==2020.1
requests==2.23.0
scipy==1.4.1
slider==0.2.1
toolz==0.10.0
urllib3==1.25.9
wtc==1.1.3
```

Reply:
Looks like an addition in `1.17` or later: <https://github.com/numpy/numpy/pull/13873>, unless I'm misinterpreting this pr.